data/wanderer/wanderer missions.txt

```diff
@@ -80,14 +80,14 @@ mission "Wanderers: Mereti: The Plant 3"
 	landing
 	name "A Home for the Plant"
 	passengers 1
-	description "Bring Tema'a Iriket to the Korath planets she marked, to determine whether any of them are a suitable home for the plant from the Mereti."
+	description "Bring Tema'a Iriket to the Korath planets she marked to determine whether any of them are a suitable home for the plant from the Mereti."
 	source "Solima Skarati"
 	destination "Sopi Lefarkata"
 	to offer
 		has "Wanderers: Mereti: The Plant 2: done"
 	on complete
 		conversation
-			`This time, Iriket guides you down into an equatorial lowland - a dry grassy plain over which a cool wind blows. You can feel its similarity to the steppe you visited on Solima Skarati, but this time you can breathe freely without the air stinging your eyes and mouth. Iriket takes some soil and places it in a small device, looking at it for a moment before turning to you. "Here. It will [grow, prosper] here."`
+			`This time, Iriket guides you down into an equatorial lowland - a dry, grassy plain over which a cool wind blows. You can feel its similarity to the steppe you visited on Solima Skarati, but this time you can breathe freely without the air stinging your eyes and mouth. Iriket takes some soil and places it in a small device, looking at it for a moment before turning to you. "Here. It will [grow, prosper] here."`
```

Comment:
```suggestion
			`This time, Iriket guides you down into an equatorial lowland - a dry, grassy plain over which a cool wind blows. You can feel its similarity to the steppe you visited on Solima Skarati, but this time you can breathe freely, without the air stinging your eyes and mouth. Iriket takes some soil and places it in a small device, looking at it for a moment before turning to you. "Here. It will [grow, prosper] here."`
```
I missed this suggestion! I naturally pause there when reading this paragraph.